### PR TITLE
Added Persian(Farsi) langauge

### PR DESCRIPTION
**Pull request by leopard20**
I don't know if this should be linked to another file to work.
Needs testing.

### CODE REVIEW

File: android/res/values-fa/strings.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <string name="title_activity_launcher">CS16-Client اجرا کننده</string>
+  <string name="launch_button">!CS16-client اجرای</string>
+  <string name="cmd_args_text">:دستورات خط فرمان</string>
```
Comment (a1batross) [inline android/res/values-fa/strings.xml]: Dude, there is no need to translate command line args. Engine doesn't know about Persian. ;)

**Comment by leopard20** [comment]
Right. Sorry. 
Will fix it in a jiffy!

**Comment by leopard20** [comment]
@a1batross 
Activate the build. I need to test it.

File: android/res/values-fa/strings.xml (hunk 1 above)
Comment (leopard20) [inline android/res/values-fa/strings.xml]: You mean:
<string name="cmd_args_text">:دستورات خط فرمان</string> ?
I'm a little confused. Command line arguments is just a string. What do you mean I shouldn't translate it?

**Comment by a1batross** [comment]
Everything is good, sorry me, I mistaken.

**Pull request merged**